packages/web-components/src/badge/badge.ts

```diff
@@ -14,7 +15,7 @@ export class Badge extends FASTBadge {
    * HTML Attribute: appearance
    */
   @attr
-  public appearance: BadgeAppearance = 'filled';
+  public appearance: BadgeProps['appearance'] = 'filled';
```

Comment:
Not proposing React dep, but showing types exist and match currently. Would like to consider how we could share a common definition. Known issues around attrs vs props types for complex data and fns.

Reply:
I think we can augment for react here to get alignment, but I'd be hesitant to take this approach w/ TS here for WC itself. I think what we want is to be able to represent the React structure as the component interface, which doesn't necessitate changing these IMO

Reply:
Imported this comment to #5